stories/theme.SLR.mdx

```diff
@@ -73,6 +73,19 @@ import { GridContainer, Grid } from '$veda-ui/@trussworks/react-uswds';
     </Figure>
 </Block>
 
+<Block type="wide">
+  <Figure>
+    <Embed
+      height="405"
+      src="https://www.youtube.com/embed/T7qcwWVMiVs"
+      title="NASA and Sea Level Rise"
+    />
+    <Caption attrAuthor="NASA" attrUrl="https://nasa.gov/">
+    The city of Mobile, AL is working with NASA's Sea Level Rise Team to plan for future infrastructure projects and to protect Mobile's coastal resources. As sea levels rise globally, coastal cities feel the effects of more frequent and more severe storms and flooding. NASA's sea level rise data, in conjunction with NOAA data, helps Mobile and other coastal communities plan for a more resilient future.
```

Comment:
Who reviewed this text? I ask because I just went through the process of swapping sea level rise to sea level change per Nadya's request. We should confirm that language should be updated here as well.